render/engine.go

```diff
@@ -54,40 +52,36 @@ func (e *Engine) ParseTemplate(templateFilename, templateDirectory string) (*tem
 	return templater.ParseGlob(fmt.Sprintf("%s/*.tmpl", templateDirectory))
 }
 
-// BuildRenderNode returns a function that fetches the appropriate template name and renders and the provided node to
-// disk.
-// TODO: This could simply be e.Render() as opposed to returning another function
-func (e *Engine) BuildRenderNode() tree.TraversalFunc {
-	return func(key string, parent, node tree.NodeTraverser) (tree.NodeTraverser, error) {
-		if s, ok := node.(*schema.Schema); ok {
-			// This allows us to avoid rendering schemas with these types
-			if s.Type != "object" && s.Type != "array" {
-				return node, nil
-			}
-		}
-
-		// find appropriate template name based on node provided
-		templateFile := GetTemplateFilename(node)
-		if templateFile == "" {
+// Render fetches the appropriate template name and renders yielded by it and the provided node to disk.
```

Comment:
Is this grammar correct? I'm struggling to grok the sentence and understand the intent.

Reply:
`renders yielded` should be `renders what is yielded`: I've made a note of this one as well